src/components/App.css

```diff
@@ -1,22 +1,39 @@
-.App {
-  text-align: center;
+.app{
+  width: 700px;
+  margin: 0 auto;
 }
 
```

Comment:
You used the correct width, but how does it fill the screen again, try to debug this 
(700px shouldn't fill the screen)

![Screen Shot 2019-12-11 at 5 18 41 PM](https://user-images.githubusercontent.com/40712459/70638648-8f3bdc80-1c39-11ea-9cb5-0d50708ed186.png)